polaris/dataset/_dataset.py

```diff
@@ -58,7 +58,7 @@ class Dataset(BaseArtifactModel):
         annotations: Each column _can be_ annotated with a [`ColumnAnnotation`][polaris.dataset.ColumnAnnotation] object.
             Importantly, this is used to annotate whether a column is a pointer column.
         source: The data source, e.g. a DOI, Github repo or URI.
-        license: The dataset license
+        license: The dataset license. Polaris only supports some Creative Commons licenses. See the SupportedLicenseType enum for accepted ID values.
```

Comment:
You can reference other code in your docstring by using double square brackets, e.g. 

```
[`SupportedLicensesTyp`][polaris.utils.types.SupportedLicenseType]
```